Linting errors

During import Ansible Galaxy is showing these errors.

```
Starting import: task_id=11300, artifact_id=11304 
Importing with galaxy-importer 0.3.0 
Getting doc strings via ansible-doc 
Finding content inside collection 
Loading module meraki_organization 
Linting module meraki_organization via flake8... 
Loading module meraki_ms_l3_interface 
Linting module meraki_ms_l3_interface via flake8... 
Loading module meraki_mr_settings 
Linting module meraki_mr_settings via flake8... 
plugins/modules/meraki_mr_settings.py:126:1: F401 'ansible.module_utils.common.dict_transformations.snake_dict_to_camel_dict' imported but unused 
Loading module meraki_config_template 
Linting module meraki_config_template via flake8... 
Loading module meraki_device 
Linting module meraki_device via flake8... 
plugins/modules/meraki_device.py:296:5: F841 local variable 'query_device_uplink_urls' is assigned to but never used 
Loading module meraki_ms_ospf 
Linting module meraki_ms_ospf via flake8... 
plugins/modules/meraki_ms_ospf.py:215:25: F841 local variable 'md5_settings' is assigned to but never used 
Loading module meraki_network 
Linting module meraki_network via flake8... 
Loading module meraki_content_filtering 
Linting module meraki_content_filtering via flake8... 
Loading module meraki_mx_site_to_site_vpn 
Linting module meraki_mx_site_to_site_vpn via flake8... 
Loading module meraki_ms_switchport 
Linting module meraki_ms_switchport via flake8... 
Loading module meraki_mr_l3_firewall 
Linting module meraki_mr_l3_firewall via flake8... 
Loading module meraki_switchport 
Linting module meraki_switchport via flake8... 
Loading module meraki_ms_storm_control 
Linting module meraki_ms_storm_control via flake8... 
Loading module meraki_mx_site_to_site_firewall 
Linting module meraki_mx_site_to_site_firewall via flake8... 
Loading module meraki_site_to_site_vpn 
Linting module meraki_site_to_site_vpn via flake8... 
Loading module meraki_mx_l3_firewall 
Linting module meraki_mx_l3_firewall via flake8... 
Loading module meraki_mx_content_filtering 
Linting module meraki_mx_content_filtering via flake8... 
Loading module meraki_mx_uplink_bandwidth 
Linting module meraki_mx_uplink_bandwidth via flake8... 
Loading module meraki_vlan 
Linting module meraki_vlan via flake8... 
Loading module meraki_nat 
Linting module meraki_nat via flake8... 
Loading module meraki_mr_ssid 
Linting module meraki_mr_ssid via flake8... 
Loading module meraki_mx_intrusion_prevention 
Linting module meraki_mx_intrusion_prevention via flake8... 
Loading module meraki_static_route 
Linting module meraki_static_route via flake8... 
Loading module meraki_mr_rf_profile 
Linting module meraki_mr_rf_profile via flake8... 
plugins/modules/meraki_mr_rf_profile.py:414:1: F401 'ansible.module_utils.common.dict_transformations.snake_dict_to_camel_dict' imported but unused 
plugins/modules/meraki_mr_rf_profile.py:416:1: F401 're.sub' imported but unused 
plugins/modules/meraki_mr_rf_profile.py:605:9: F841 local variable 'profile' is assigned to but never used 
Loading module meraki_switch_storm_control 
Linting module meraki_switch_storm_control via flake8... 
Loading module meraki_switch_stack 
Linting module meraki_switch_stack via flake8... 
Loading module meraki_mx_malware 
Linting module meraki_mx_malware via flake8... 
Loading module meraki_mx_vlan 
Linting module meraki_mx_vlan via flake8... 
Loading module meraki_firewalled_services 
Linting module meraki_firewalled_services via flake8... 
Loading module meraki_webhook 
Linting module meraki_webhook via flake8... 
plugins/modules/meraki_webhook.py:196:5: F841 local variable 'e' is assigned to but never used 
Loading module meraki_admin 
Linting module meraki_admin via flake8... 
Loading module meraki_mx_l2_interface 
Linting module meraki_mx_l2_interface via flake8... 
plugins/modules/meraki_mx_l2_interface.py:166:1: F401 'ansible.module_utils.common.dict_transformations.recursive_diff' imported but unused 
Loading module meraki_mx_nat 
Linting module meraki_mx_nat via flake8... 
Loading module meraki_intrusion_prevention 
Linting module meraki_intrusion_prevention via flake8... 
Loading module meraki_management_interface 
Linting module meraki_management_interface via flake8... 
Loading module meraki_alert 
Linting module meraki_alert via flake8... 
Loading module meraki_ssid 
Linting module meraki_ssid via flake8... 
Loading module meraki_ms_access_list 
Linting module meraki_ms_access_list via flake8... 
Loading module meraki_ms_link_aggregation 
Linting module meraki_ms_link_aggregation via flake8... 
Loading module meraki_switch_access_list 
Linting module meraki_switch_access_list via flake8... 
Loading module meraki_ms_stack 
Linting module meraki_ms_stack via flake8... 
Loading module meraki_mx_l7_firewall 
Linting module meraki_mx_l7_firewall via flake8... 
Loading module meraki_malware 
Linting module meraki_malware via flake8... 
Loading module meraki_mx_uplink 
Linting module meraki_mx_uplink via flake8... 
Loading module meraki_syslog 
Linting module meraki_syslog via flake8... 
Loading module meraki_mx_static_route 
Linting module meraki_mx_static_route via flake8... 
Loading module meraki_snmp 
Linting module meraki_snmp via flake8... 
Loading module_utils network.meraki.meraki 
Linting module_utils network.meraki.meraki via flake8... 
plugins/module_utils/network/meraki/meraki.py:146:21: F841 local variable 'new' is assigned to but never used 
Loading doc_fragments meraki 
Linting doc_fragments meraki via flake8... 
Collection loading complete 
Processing via galaxy-importer complete 
Checking dependencies in importer data 
Publishing collection 
Collection published 
Import completed with 9 warnings and 0 errors 
```

--- a/changelogs/fragments/sanity-fixes.yml
+++ b/changelogs/fragments/sanity-fixes.yml
@@ -1,0 +1,3 @@
+---
+bugfixes:
+  - Fix some flake8 sanity errors as reported by Ansible Galaxy. Should be no functional change.

